### DISCUSSION TITLE
能在堆栈窗口显示api参数吗？

### DISCUSSION THREAD

**Discussion by x7dbg** [x64dbg/x64dbg#3065] [Q&A]
效果od那样

**Comment by zhiyoutech** [comment]
您好，我也在研究x64dbg 的GUI部分。我尝试使用QT6.6编译此GUI，但是，编译过程报出很多问题，比如agents.h没有找到。请问，方便留下联系方式与你进一步沟通吗？我的QQ:641464726@qq.com

**Comment by torusrxxx** [comment]
不能（#2552）
